src/crypto/tls/handshake_server_tls13.go

```diff
@@ -815,9 +895,21 @@ func (hs *serverHandshakeStateTLS13) readClientCertificate() error {
 			c.sendAlert(alertIllegalParameter)
 			return errors.New("tls: client certificate used with invalid signature algorithm")
 		}
+
+		if certMsg.delegatedCredential {
+			if err := hs.processDelegatedCredentialFromClient(certMsg.certificate.DelegatedCredential, certVerify); err != nil {
+				// TODO: chris, how should we report the error?
+				return err
```

Comment:
a TODO here

Reply:
Yes, I'm expecting @chris-wood to answer this in his review ;)

Reply:
Shouldn't this be like any other failed client authentication?

Reply:
I returned the error for the moment. @chris-wood , let me know your thoughts